src/sign/index.ts

```diff
@@ -53,7 +53,7 @@ export default class Client {
     if (!message.timestamp)
       message.timestamp = parseInt((Date.now() / 1e3).toFixed());
     const data: any = { domain, types, message };
-    const sig = await signer._signTypedData(domain, data.types, message);
+    const sig = await signer?._signTypedData(domain, data.types, message);
```

Comment:
Why do we get signer undefined? 🤔 what should happen if it is undefined? 
If it is like this we send empty sig to the hub, right? maybe would be good to show some error message?

Reply:
I can't reproduce signer undefined. Anytime I log the signer it's available. Not sure why it's showing this error.

Reply:
Ok can reproduce now.